src/requirements/requirement-graph-builder.ts

```diff
@@ -12,86 +18,97 @@ type BuildRequirementFulfillmentGraphParameters<Requirement extends string, Cour
   readonly userCourses: readonly Course[];
   /**
    * Some requirements might have several different ways to fulfill them. This is a map from
-   * toggleable requirements to a list of courses that could be applied to the requirement under
+   * toggleable requirements to a list of course IDs that could be applied to the requirement under
```

Comment:
This is the crseId (that crosslisted courses share) and not unique ID right?

Reply:
Yes, because it's used to match to eligible courses.